packages/components/accessible-hidden/README.md

```diff
@@ -0,0 +1,50 @@
+# Accessible Hidden
+
+#### Description
+
+This component is used to hide content offscreen, removing it from sighted users, while keeping the still accessible to screenreaders and other assistive technology.
+It can also be useful for testing with tools like react-testing-library and cypress, which query elements through content which you might not want to be showing explicitly, such as querying for a input by its label while the label is visually hidden.
```

Comment:
```suggestion
It can also be useful for testing with tools like react-testing-library and cypress, which requires quering elements through content which might not be intended to be visible on screen, such as querying for a input by its label while the label is visually hidden.
```